src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String COLOUR = "white";
+    private static final int RADIUS = 10;
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomFigureSide = random.nextInt(13);

```

Comment:
make 13 a constant variable, but `randomFigureSide` should  be the local variable